public/app/themes/justice/inc/admin.php

```diff
@@ -36,4 +37,41 @@ public static function removeCustomizer()
         $customizer_url = add_query_arg('return', urlencode(remove_query_arg(wp_removable_query_args(), wp_unslash($_SERVER['REQUEST_URI']))), 'customize.php');
         remove_submenu_page('themes.php', $customizer_url);
     }
+
+    public static function removePublishPressUpsellBranding()
+    {
+        echo '<style>
+               .toplevel_page_revisionary-q {
+                 li:last-of-type {
+                   display: none;
+                 }
+               }
+               .pp-version-notice-bold-purple {
+                 display: none;
+               }
+               .pressshack-admin-wrapper {
+                 a {
+                  color: #2271b1;
+                  &:hover {
+                    color: #135e96;
+                  }
+                  &:focus {
+                    color: #043959;
+                  }
+                }
+                 .inner-sidebar {
+                   display: none;
+                 }
+                 footer {
+                   display: none;
+                 }  
+               }
+               .ppseries-settings-body-content {
+                 width: 100% !important;
+               }
+               .rvy-creation-ui {
+                 display: none;
+               }
```

Comment:
^ I'm hiding the UI elements in the visual editor because the preview doesn't work (it seems to be a bug in the plugin).

There are buttons in the admin interface that do work so this shouldn't be an issue.

Reply:
I see, do you mind sharing steps to reproduce? It might be worth fixing the bug.

See [my comment](https://github.com/ministryofjustice/justice-gov-uk/pull/115#discussion_r1588955479) about the unintended consequence of this rule.

Reply:
Sure, steps to reproduce are: 

1. Edit a page and click New Revision.
![Screenshot 2024-05-03 at 14 03 05](https://github.com/ministryofjustice/justice-gov-uk/assets/24455584/f8ff6aad-275f-4269-a847-37e13b3a5374)
2. Click the Preview or Edit button that appears.
![Screenshot 2024-05-03 at 14 04 33](https://github.com/ministryofjustice/justice-gov-uk/assets/24455584/2234cb7b-5da7-46ba-b9be-9becdfbddfd6)
3. You will be redirected to the wrong page and see either a white screen or the 'Headers already sent' error message.

The preview and edit buttons in the revision queue work correctly.

Reply:
Hey @EmilyHazlehurst , I had a poke around and found the reason for the bug.

The plugin was sending redirect headers after the html had started rendering.

I've changed the hooks to ones that fire before the html starts to render, would you mind copy pasting the bit of code in this PR to your local just to verify that it works. Thanks :)

https://github.com/publishpress/PublishPress-Revisions/pull/1238

Reply:
Thanks @EarthlingDavey, that's now in the plugin and works perfectly when updated :)